src/SectorView.cpp

```diff
@@ -47,6 +47,145 @@ enum DetailSelection {
 static const float ZOOM_SPEED = 15;
 static const float WHEEL_SENSITIVITY = .03f; // Should be a variable in user settings.
 
+#define FFRAC(_x) ((_x)-floor(_x))
+
+enum Side {
+	XMIN,
+	XMAX,
+	YMIN,
+	YMAX,
+	ZMIN,
+	ZMAX,
+	SIDES
+};
+
+using box_t = std::array<int, SIDES>;
+using cache_t = SectorCache::Slave *;
```

Comment:
This is a problem. `cache_t` hides the fact that this is a pointer type to an already typedef-ed type alias (`SectorCache` being a wrapper typedef for a long template). It's only used in three functions, and the actual type name is fairly short. I'm sorry I didn't catch it in the initial review, I was focusing on the box_t above it. In general, there's no need to typedef this, especially not to a "C-style" typename - we're not writing kernel-style opaque C interfaces here where a `cache_t` is a pointer to an opaque anonymous struct typedef that is never defined in the interface.

Reply:
Done!